docs/content/latest/secure/audit-logging/audit-logging-ysql.md

```diff
@@ -182,8 +183,6 @@ Start the YugabyteDB Cluster with the following Audit logging Configuration.
     --ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice,pgaudit.log_client=ON"
```

Comment:
The output looks wrong here, I guess due to the indenting; also the code blocks need to specify output or sql or sh, depending

Reply:
Fixed the file